app/src/main/java/org/onionshare/android/tor/OnionService.kt

```diff
@@ -0,0 +1,38 @@
+package org.onionshare.android.tor
+
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import dagger.hilt.android.AndroidEntryPoint
+import org.onionshare.android.ui.NOTIFICATION_ID
+import org.onionshare.android.ui.OnionNotificationManager
+import org.slf4j.LoggerFactory.getLogger
+import org.torproject.jni.TorService
+import javax.inject.Inject
+
+private val LOG = getLogger(OnionService::class.java)
+
+@AndroidEntryPoint
+class OnionService : TorService() {
```

Comment:
Shall we merge this and then go back and consider how to handle Hans' suggested architecture where the foreground service lives in the main process and binds TorService?

Reply:
Yes, let's do this in the next MR when you run Tor in its own process.